### ISSUE TITLE
Training Loss/ Accuracy

### ISSUE THREAD

**Issue by sudhanshu22gupta** [JordanCola/Facial-Recognition-VGG-Face#3]
I am unable to see any improvement in accuracy or loss in the training. It remains the same throughout the network: loss: 0.6931 - accuracy: 0.5000 - val_loss: 0.6931 - val_accuracy: 0.5000

Am I doing something wrong? I haven't made any changes to your data set or code.

**Comment by Rigzbot** [comment]
Hi @sudhanshu22gupta I am facing the same error, did you get any solution?
